backend/templates/impressum.html

```diff
@@ -29,8 +21,7 @@ <h1>Impressum</h1>
                     <h5>Personen hinter dem Projekt</h5>
                     <strong>match4everyone e.V.</strong><br><br>
                     Tel.: ---- <br>
-                    E-Mail:
-                    <script type="text/javascript">document.write(decode("abc@example.qr"))</script>
+                    E-Mail: "abc@example.qr"
```

Comment:
Why do you remove the spam protection?

Reply:
If we feel we need that, we can do it properly later. We can make an issue. But this code is just plain awful and undocumented. We always have the old repo to bring it back.

Reply:
I would not remove features because the code is undocumented. Do it the other way round; open an issue to improve this and leave it in like this

Reply:
Revert 252692f8188ada8b4a63515c056d51fee2ec7b57